general/cisetup/cisetup.go

```diff
@@ -656,16 +655,17 @@ func getExplicitTechsListByNumber(techs []string) string {
 	return strings.Join(techs[0:len(techs)-1], ", ") + " and " + techs[len(techs)-1]
 }
 
-func (cc *CiSetupCommand) getBuildCmd(tech cisetup.Technology) {
-	defaultBuildCmd := buildCmdByTech[tech]
-	if info, built := cc.defaultData.BuiltTechnologies[tech]; built {
-		if info.BuildCmd != "" {
-			defaultBuildCmd = info.BuildCmd
+func (cc *CiSetupCommand) getBuildCmd() {
+	defaultBuildCmd := buildCmdByTech[cc.data.BuiltTechnology.Type]
+	// Use the cached build command only if the chosen built technology wasn't changed.
+	if cc.defaultData.BuiltTechnology.Type == cc.data.BuiltTechnology.Type {
+		if cc.data.BuiltTechnology.BuildCmd != "" {
```

Comment:
If pretty sure this if condition should be on `cc.defaultData` (the cached build cmd)